apps/condo/domains/banking/access/BankIntegrationAccessRight.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Generated by `createschema banking.BankIntegrationAccessRight 'integration:Relationship:BankIntegration:PROTECT; user:Relationship:User:CASCADE;'`
+ */
+
+const { userIsOwner } = require('@open-condo/keystone/access')
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+
+const { BankIntegrationAccessRight } = require('@condo/domains/banking/utils/serverSchema')
+
+async function canReadBankIntegrationAccessRights (args) {
+    const { authentication: { item: user } } = args
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin || user.isSupport) return {}
+    if (userIsOwner(args)) return {}
```

Comment:
If we need to allow reading only records, belonging to current user, then just
```suggestion
    return { user: { id: user.id } }
```